lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.html

```diff
@@ -28,7 +28,8 @@ <h3 class="adf-inherit-container-header">
                         aria-label="{{'PERMISSION_MANAGER.LABELS.INHERITED_PERMISSION_TOGGLE' | translate}}"
                         data-automation-id="adf-inherit-toggle-button"
                         [checked]="model.node.permissions.isInheritanceEnabled"
-                        (change)="permissionList.toggleInherited($event)" />
+                        (change)="permissionList.toggleInherited($event)"
+                        *ngIf="updatePermissionsAllowed" />
```

Comment:
It's small change but maybe we could move `*ngIf` to the top of the element? So that it will be easier to see that element has some rendering condition applied for the visibility

Reply:
done